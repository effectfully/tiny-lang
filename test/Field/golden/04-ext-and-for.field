ext i : field;
assert i > 0;
for i = 1 to 2 do
    assert i > 0;
end;
assert i > 0;
