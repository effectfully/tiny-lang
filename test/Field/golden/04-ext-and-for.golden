ext i_0 : field;
assert (i_0 : field) > 0;
let i_1 : field = 1;
assert (i_1 : field) > 0;
let i_2 : field = 2;
assert (i_2 : field) > 0;
assert (i_2 : field) > 0;
